app/app.controller.js

```diff
@@ -9,8 +9,13 @@ res.status(200).send({ topics})
 }
 
 exports.getApiEndpoints = (req, res, next) => {
-selectApiEndpoints().then((body) => {
-res.status(200).json(body)
-})
-.catch(next)
+
+    const endpointsFilePath =  '/Users/ericnabofa/northcoders/projects/be-nc-news/endpoints.json'
```

Comment:
This path is not dynamic so will not work if this repo is used on another machine (eg when hosted) you could use __dirname or follow my hint above to just require the file in instead of using any fs functionality 👍

Reply:
done